app/jobs/sample_manifest/generate_wells_job.rb

```diff
@@ -1,25 +1,31 @@
 # frozen_string_literal: true
-# Generates wells for plate sample manifests
+# Generates wells and sample manifest assets, for a plate sample manifest.
 SampleManifest::GenerateWellsJob =
-  Struct.new(:sample_manifest_id, :map_ids_to_sample_ids, :plate_id) do
+  Struct.new(:sample_manifest_id, :map_ids_to_sanger_sample_ids, :plate_id) do
     def perform
       ActiveRecord::Base.transaction do
-        # Ensure the order of the wells are maintained
-        maps = Map.find(map_ids).index_by(&:id)
-        well_data = map_ids_to_sample_ids.map { |map_id, sample_id| [maps[map_id], sample_id] }
+        map_ids_to_sanger_sample_ids.each { |k, v| create_well(k, v) }
```

Comment:
@KatyTaylor Would it be worth identifying why CodeClimate feels the need to tell you about this?  I feel it's because there's a flag to warn of single character variable names.  Not that it tells you this in the error message, or suggest how you'd "fix" this.

Reply:
Think it's this - https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md

I've changed it, although the reason I did it was I thought it was fairly obvious what the key and value would be, since `map_ids_to_sanger_sample_ids` on the same line - and it makes it a lot more wordy.

Reply:
I agree that making it more wordy feels unnecessary and might warrant us looking into whether we're getting value from this check, or if we can prevent it triggering when we don't want it to.  It's hard to get good use out of a tool that fires off too many false positives.